genshin/models/genshin/chronicle/notes.py

```diff
@@ -73,6 +73,49 @@ def seconds(self) -> int:
         return self.timedata[3]
 
 
+class TaskReward(APIModel):
+    """Status of the Commission/Task.
+
+    status:
+        TaskRewardStatusTakenAward
+        TaskRewardStatusUnfinished
+    """
+
+    status: str
+
+
+class AttendanceReward(APIModel):
+    """Status of the Encounter Point.
+
+    status:
+        AttendanceRewardStatusForbid
+        AttendanceRewardStatusTakenAward
+        AttendanceRewardStatusWaitTaken
+        AttendanceRewardStatusUnfinished
+
+    progress:
+        2000 == 100%
+        1000 == 50%
+    """
+
+    status: str
+    progress: int
+
+
+class DailyTasks(APIModel):
+    """Daily tasks section."""
+
+    max_tasks: int = Aliased("total_num")
+    completed_tasks: int = Aliased("finished_num")
+    claimed_commission_reward: bool = Aliased("is_extra_task_reward_received")
```

Comment:
So they are now duplicated like `{"total_num": 4, "daily_tasks": {"total_num": 4, ...}, ...}`?

Reply:
That is correct